sky/packages/sky/lib/src/rendering/paragraph.dart

```diff
@@ -15,59 +15,66 @@ export 'package:sky/src/painting/text_painter.dart';
 // differently. To work around this problem for now, we round fractional pixel
 // values up to the nearest whole pixel value. The right long-term fix is to do
 // layout using fixed precision arithmetic.
-double _applyFloatingPointHack(double layoutValue) {
+double applyFloatingPointHack(double layoutValue) {
   return layoutValue.ceilToDouble();
 }
 
 /// A render object that displays a paragraph of text
 class RenderParagraph extends RenderBox {
 
-  RenderParagraph(TextSpan text) : _textPainter = new TextPainter(text) {
+  RenderParagraph(
+      TextSpan text,
```

Comment:
Indent two characters, not four. Put spaces around the named arguments.

(See the style guide for details.)